psd-web/spec/models/user_spec.rb

```diff
@@ -1,6 +1,71 @@
 require "rails_helper"
 
 RSpec.describe User do
+  describe "validations" do
+    context "on registration completion" do
+      it "validates the presence of the mobile number" do
+        user = build(:user, mobile_number: "")
+        expect(user.valid?(:registration_completion)).to eq false
```

Comment:
RSpec provides a nice helper for this:
```suggestion
        expect(user).to be_valid(:registration_completion)
```